docs/current_docs/manuals/developer/cookbook.mdx

```diff
@@ -589,18 +589,96 @@ The following Dagger Function tags a container image multiple times and publishe
 
 #### Examples
 
-- Tag and publish a container image to Docker Hub, using the account username `user` and the password set in the `PASSWORD` environment variable:
+- Tag and publish a just-in-time container image to Docker Hub, using the account username `user` and the password set in the `PASSWORD` environment variable:
 
     ```shell
     dagger call publish --registry=docker.io --username=user --password=env:PASSWORD
     ```
 
-- Tag and publish a container image to GitHub Container Registry, using the account username `user` and the GitHub personal access token set in the `PASSWORD` environment variable:
+- Tag and publish a just-in-time container image to GitHub Container Registry, using the account username `user` and the GitHub personal access token set in the `PASSWORD` environment variable:
 
     ```shell
     dagger call publish --registry=ghcr.io --username=user --password=env:PASSWORD
     ```
 
+### Export a directory or file to the host
+
+The following Dagger Functions return a just-in-time directory and file. These outputs can be exported to the host with `dagger call ... export ...`.
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./cookbook/snippets/export-file-dir/go/main.go
+```
+
+</TabItem>
+<TabItem value="Python">
+
+```python file=./cookbook/snippets/export-file-dir/python/main.py
+```
+
+</TabItem>
+<TabItem value="TypeScript">
+
+```typescript file=./cookbook/snippets/export-file-dir/typescript/index.ts
+```
+
+</TabItem>
+</Tabs>
+
+#### Examples
+
+- Export the directory returned by the Dagger Function to the `/home/admin/export` path on the host:
+
+    ```shell
+    dagger call \
+      get-dir \
+      export --path=/home/admin/export
+    ```
+
+- Export the file returned by the Dagger Function to the `/home/admin/myfile` path on the host:
+
+    ```shell
+    dagger call \
+      get-file \
+      export --path=/home/admin/myfile
+    ```
+
+### Export a container image to the host
```

Comment:
In this example it might be useful to show how you can also `docker load` it and run it in your docker engine

Reply:
Added